lib/views/create-bug.js

```diff
@@ -157,7 +152,7 @@ function submitBug() {
 
   this.app.bugzilla.createBug({
     product: 'Firefox OS',
-    component: value('#component'),
+    component: (process.env.TEST ? 'GAIA' : value('#component')),
```

Comment:
When you start the app with `TEST=1 npm start` we run against https://bugzilla-dev.allizom.org/, this doesnt have a Gaia::Feedback component, so ignore the default values and just hardcode to Gaia

(this is the correct version of the one I backed out previously)